tests/ha/test_multi_server_pod_failure.py

```diff
@@ -629,3 +630,484 @@ def test_writes_till_kserver_pods_fail(self):
 
         LOGGER.info("Completed: Test to verify degraded WRITEs after each pod failure till K "
                     "server pods down - unsafe shutdown.")
+
+    # pylint: disable=C0321
+    # pylint: disable=too-many-locals
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-40577")
+    @CTFailOn(error_handler)
+    def test_ios_during_kserver_pods_down(self):
+        """
+        This test tests continuous READs/WRITEs/DELETEs while pods are failing till K
+        server pods down - unsafe shutdown
+        """
+        LOGGER.info("STARTED: Test to verify continuous READs/WRITEs/DELETEs while pods are "
+                    "failing till K server pods down - unsafe shutdown.")
+
+        event = threading.Event()  # Event to be used to send when server pods going down
+        wr_bucket = self.kvalue * 5 + HA_CFG["s3_bucket_data"]["no_buckets_for_deg_deletes"]
```

Comment:
we don't need this logic here. 140 buckets would be enough for background deletes. as we are doing 150-10

Reply:
For now I will keep as it is